### DISCUSSION TITLE
Wikipathways download site URL

### DISCUSSION THREAD

**Discussion by risserlin** [wikipathways/wikipathways-help#79] [Q&A] [answered]
On a monthly basis I pull wikipathways data and incorporate it into our conglomerate gmt files.  the URL changed to https://wikipathways-data.vmcloud.org/current/gmt in June 2023 (according to my notes in my scripts.  Might have happened before that but that was when I caught it and updated it).  the wikipathways failed this past month and I noticed the url was changed back to https://data.wikipathways.org/current/gmt/ which I was using previously.  
I was just wondering if the previous change was a temporary change or was it an accidental change back to the previous address recently?  going forward which URL should I use.
Thanks, 
Ruth

**Comment by khanspers** [comment, marked as answer]
Hi @risserlin. We just switched to a new system this month. Going forward, please use https://data.wikipathways.org/current/gmt/.

**Comment by AlexanderPico** [comment]
Right. The data.wikipathways.org has been and will continue to be the correct URL. It previously redirected to vmcloud, which we no longer use. In June, there must have been a temporary issue with the redirect, which led you to the vmcloud URL as reasonable fix.

Sorry for the hassle!
